administrator/components/com_admin/script.php

```diff
@@ -7336,13 +7338,19 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
 
 		foreach ($folders as $folder)
 		{
-			if ($folderExists = Folder::exists(JPATH_ROOT . $folder))
+			$folderPath = JPATH_ROOT . $folder;
+
+			if ($folderExists = Folder::exists($folderPath))
 			{
 				$status['folders_exist'][] = $folder;
 
-				if ($dryRun === false)
+				if (!(Folder::isEmpty($folderPath)))
```

Comment:
```suggestion
				if (!Folder::isEmpty($folderPath))
```
Remove obsolete brackets